lib/dsl/lexer.js

```diff
@@ -23,8 +23,51 @@ const Lexer = chevrotain.Lexer;
 
 const tokens = {};
 
+const baseNamePattern = 'baseName';
```

Comment:
It's not really a pattern... its use is, but it's not. The name is kinda misleading.

Reply:
Yes I see your point